static/js/app.js

```diff
@@ -770,7 +780,14 @@ $(function () {
             var center = app.map.getCenter();
             return "start_date=" + moment(this.dateRanges[0]).format("YYYY-MM-DD") +
                 "&end_date=" + moment(this.dateRanges[1]).format("YYYY-MM-DD") +
-                "&zoom=" + app.map.zoom + "&lat=" + center.lat() + "&lon=" + center.lng();
```

Comment:
This is starting to get messy. Perhaps we can do something like:

    for attr in ATTRS:
        url += "&%s=%s" % (attr, getattr(this, attr))